discord/activity.py

```diff
@@ -769,7 +774,7 @@ def __init__(
     ):
         super().__init__(**extra)
         self.name: str | None = name
-        self.state: str | None = extra.pop("state", None)
+        self.state: str | None = extra.pop("state", name)
         if self.name == "Custom Status":
             self.name = self.state
```

Comment:
sure about that?

Reply:
Sure about L777, yeah, but L778-779 is out of the scope of this pr.